src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -206,6 +206,10 @@ message PageStreamingConfigProto {
 message PageStreamingRequestProto {
   // The name of the field in the request containing the page token.
   string token_field = 1;
+
+  // The name of the field in the request containing the desired number of items
+  // in the next page.
+  string page_size_field = 2;
```

Comment:
Right, I did make a change to the config generator - I checked your pull and the implementation seems to be _nearly_ identical, so should be fine